theLudovyc/TheMarket.gd

```diff
@@ -21,11 +21,13 @@ func _ready():
 
 
 func get_resource_cost(resource_type: Resources.Types) -> int:
-	if not Resources.Levels.has(resource_type):
-		# ERROR
-		return 0
-
-	return Resources.Levels[resource_type] + 1
+	#TODO Michael: check this method
```

Comment:
#TODO Michael: check this method

Reply:
Adjusted method to current implementation.